### PR TITLE
Refactor changes due to Ballerina Streams grammar improvements

### PR DESCRIPTION
**Pull request by mohanvive**

### CODE REVIEW

**Comment by mohanvive** [comment]
Please don't merge this PR. A related discussion is going on in the ballerina-dev main thread [1].
[1] Avoid using keywords as the native function name

File: compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/antlr4/BallerinaParserListener.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Generated from BallerinaParser.g4 by ANTLR 4.5.3
+// Generated from /home/mohan/ballerina/git-new/grammar-fix/ballerina/compiler/ballerina-lang/src/main/resources/grammar/BallerinaParser.g4 by ANTLR 4.5.3
```
Comment (djkevincr) [inline compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/antlr4/BallerinaParserListener.java]: Shall we remove these generated file header, similarly for other generated classes?

Comment (mohanvive) [reply compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/antlr4/BallerinaParserListener.java]: @djkevincr , sure will do. There are some more refactoring pending as well.

File: compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/antlr4/BallerinaLexer.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Generated from BallerinaLexer.g4 by ANTLR 4.5.3
+// Generated from /home/mohan/ballerina/git-new/grammar-fix/ballerina/compiler/ballerina-lang/src/main/resources/grammar/BallerinaLexer.g4 by ANTLR 4.5.3
```
Comment (anupama-pathirage) [inline compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/antlr4/BallerinaLexer.java]: Shall we remove the path here?

File: compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/antlr4/BallerinaParser.java [hunk 3]
```diff
@@ -18,79 +18,92 @@
...
+		FUNCTION=9, STREAMLET=10, CONNECTOR=11, ACTION=12, STRUCT=13, ANNOTATION=14, 
+		ENUM=15, PARAMETER=16, CONST=17, TRANSFORMER=18, WORKER=19, ENDPOINT=20, 
+		XMLNS=21, RETURNS=22, VERSION=23, FROM=24, ON=25, SELECT=26, GROUP=27, 
+		BY=28, HAVING=29, ORDER=30, WHERE=31, FOLLOWED=32, INSERT=33, INTO=34, 
+		UPDATE=35, DELETE=36, SET=37, FOR=38, WINDOW=39, QUERY=40, TYPE_INT=41, 
+		TYPE_FLOAT=42, TYPE_BOOL=43, TYPE_STRING=44, TYPE_BLOB=45, TYPE_MAP=46, 
+		TYPE_JSON=47, TYPE_XML=48, TYPE_TABLE=49, TYPE_STREAM=50, TYPE_AGGREGTION=51, 
+		TYPE_ANY=52, TYPE_TYPE=53, VAR=54, CREATE=55, ATTACH=56, IF=57, ELSE=58, 
+		FOREACH=59, WHILE=60, NEXT=61, BREAK=62, FORK=63, JOIN=64, SOME=65, ALL=66, 
+		TIMEOUT=67, TRY=68, CATCH=69, FINALLY=70, THROW=71, RETURN=72, TRANSACTION=73,
```
Comment (anupama-pathirage) [inline compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/antlr4/BallerinaParser.java]: Shall we remove the path here?

File: compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/antlr4/BallerinaParser.java [hunk 4]
```diff
@@ -1,4 +1,4 @@
-// Generated from BallerinaParser.g4 by ANTLR 4.5.3
+// Generated from /home/mohan/ballerina/git-new/grammar-fix/ballerina/compiler/ballerina-lang/src/main/resources/grammar/BallerinaParser.g4 by ANTLR 4.5.3
```
Comment (anupama-pathirage) [inline compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/antlr4/BallerinaParser.java]: Shall we remove the path here? Check all the generated files

**Comment by mohanvive** [comment]
Closing the PR since it is no longer required..

**Pull request closed without merging**